packages/flutter/test/widgets/table_test.dart

```diff
@@ -67,6 +68,41 @@ void main() {
     await run(TextDirection.rtl);
   });
 
+  testWidgets('Table widget can be detached and re-attached', (WidgetTester tester) async {
+    final Widget table = Table(
+      key: GlobalKey(),
+      children: [
+        TableRow(
+            decoration: BoxDecoration(
+                color: Colors.yellow
+            ),
+            children: const <Widget>[Placeholder()]
```

Comment:
Add a trailing comma after this and the next line and then fix indentation (only 2 spaces)